cmd/install/install.go

```diff
@@ -0,0 +1,107 @@
+package install
+
+import (
+	"fmt"
+
+	"github.com/spf13/afero"
+
+	"github.com/puppetlabs/pdkgo/pkg/install"
+	"github.com/puppetlabs/pdkgo/pkg/telemetry"
+	"github.com/puppetlabs/prm/pkg/prm"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+type InstallCommand struct {
+	ToolPkgPath  string
+	InstallPath  string
+	Force        bool
+	PctInstaller install.InstallerI
```

Comment:
Minor issue - perhaps we want to rename this param:
```suggestion
	PrmInstaller install.InstallerI
```